controllers/apply-next/simple/form.js

```diff
@@ -555,22 +556,49 @@ module.exports = function({ locale, data = {} }) {
         }
     ];
 
-    const schema = Joi.object(
-        reduce(
-            fields,
-            function(acc, field) {
-                acc[field.name] = field.schema;
-                return acc;
+    function schema() {
+        const computed = Joi.object(
+            reduce(
+                fields,
+                function(acc, field) {
+                    acc[field.name] = field.schema;
+                    return acc;
+                },
+                {}
+            )
+        );
+
+        return computed;
+    }
+
+    function summary() {
```

Comment:
Adds a summary method to the form model. This function provides a summary of the current application, rather than hard-coding assumptions about the form into the template or database model.